lib/beaker/tasks/quick_start.rb

```diff
@@ -1,49 +1,85 @@
 require 'beaker-hostgenerator'
 
+VAGRANT  = BeakerHostGenerator::CLI.new(['ubuntu1404-64default.mdcal', '--hypervisor=vagrant',
+                                         '--global-config={box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm'])
+VMPOOLER = BeakerHostGenerator::CLI.new(['redhat7-64default.mdcal-redhat7-64af'])
+
 namespace :beaker_quickstart do
 
-  desc 'Generate Default Beaker Host Config File'
-  task :gen_hosts do
-    cli = BeakerHostGenerator::CLI.new(['redhat7-64default.mdcal-redhat7-64af'])
+  desc 'Generate Default Beaker Host Config File, valid options are: vmpooler or vagrant.'
+  task :gen_hosts, [:provisioner] do |t, args|
+    hosts_file = "acceptance/config/default__#{args[:provisioner]}_hosts.yaml"
+    if args[:provisioner] == 'vagrant'
+      cli = VAGRANT
+    elsif args[:provisioner] == 'vmpooler'
+      cli = VMPOOLER
+    else
+      puts "No provisioner provided, defaulting to vmpooler."
+      cli = VMPOOLER
+    end
     FileUtils.mkdir_p('acceptance/config') # -p ignores when dir already exists
-    File.open("acceptance/config/default_hosts.yaml", 'w') do |fh|
-      fh.print(cli.execute)
+    if File.exist?(hosts_file) then
+      File.open(hosts_file, 'w') do |fh|
+        fh.print(cli.execute)
+      end
+    else
+      puts "Host Config File: #{hosts_file} already exists so not overwriting"
```

Comment:
@pinkypie is `puts` the best option for writing out in these files? Does Rake itself provide any functionality that we could hook into to manage this better?